lib/msf/core/db_manager.rb

```diff
@@ -153,9 +157,6 @@ def check
   #
   def initialize_database_support
     begin
-      # Database drivers can reset our KCODE, do not let them
-      $KCODE = 'NONE' if RUBY_VERSION =~ /^1\.8\./
```

Comment:
Was this just removed because we no longer support Ruby 1.8?

Reply:
It's unfortunate `KCODE` doesn't mean Konami Code.

Reply:
@jbarnett-r7 Yes

Reply:
@wvu-r7 :arrow_up::arrow_up::arrow_down::arrow_down::arrow_left::arrow_right::arrow_left::arrow_right::b::a: